.github/workflows/dependency-tests.yml

```diff
@@ -62,7 +60,7 @@ jobs:
         platform: [ubuntu-latest, macos-latest, windows-latest]
     runs-on: ${{ matrix.platform }}
     steps:
-      - uses: lf-lang/vscode-lingua-franca/.github/actions/build@main
+      - uses: lf-lang/vscode-lingua-franca/.github/actions/build@update-error-message
```

Comment:
Can we get ride of the `@ref` by referencing the workflow locally, like we do in `lingua-franca` now?